### ISSUE TITLE
Mod was magically deleted

### ISSUE THREAD

**Issue by ArtemaPyxis** [opera-gaming/gxmods#114]
I made and uploaded my own mod. Been working just fine for a few weeks now. I turn off my computer and go to bed like usual. Wake up in the morning, open it up, and boom, GX mod has been deleted from the store. No reason, no warning, I don't even know why it was removed or how. I wasn't sent anything in my emails, so if it was an ownership thing (my mod was based off a game I liked) I wouldn't even know. 

Went into my files and now I'm trying to reupload it. Its the same exact files, no issues there, but when I post it to the public, then go and search for it in the store, it just doesn't pop up. 

TLDR; mod was magically deleted overnight, no warnings or statements as to why, can't reupload it for some reason.

I'm a bit crushed. It's not hard to make a GX mod but its my first ever sort of coding project and I was super proud of it. It was the only mod for the game too, gave the community something nice and I watched the installs count rise day by day. I don't even know why, if I can fix it, or even how to fix it.
